.github/workflows/ci.yml

```diff
@@ -0,0 +1,23 @@
+name: Linux Local build and test
```

Comment:
nit: I recommend using simpler names (with no spaces), and potentially matching the yaml file name -> `CI`